sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -453,40 +439,26 @@ case class Expm1(child: Expression) extends UnaryMathExpression(StrictMath.expm1
   examples = """
     Examples:
       > SELECT _FUNC_(-0.1);
-       -1
+       -1.0
       > SELECT _FUNC_(5);
        5
+      > SELECT _FUNC_(3.1411, 3);
+       3.141
+      > SELECT _FUNC_(3.1411, -3);
+       1000.0
```

Comment:
why is it 1000?

Reply:
If scale_expr is negative, then it specifies the number of places before the decimal point to which to adjust the number. For example, if the scale is -2, then the result is a multiple of 100.

REF: https://docs.snowflake.com/en/sql-reference/functions/ceil.html